### ISSUE TITLE
Fallback support for proxy dashboard

### ISSUE THREAD

**Issue by Clindbergh** [lando/lando#1588]
On Windows 10, my lando failed to start:

```
$ lando start
Let's get this party started! Starting app..
Starting landoproxyhyperion5000gandalfedition_proxy_1 ... error

ERROR: for landoproxyhyperion5000gandalfedition_proxy_1  Cannot start service proxy: driver failed programming external connectivity on endpoint landoproxyhyperion5000gandalfedition_proxy_1 (f9fbb75a36e51dfaa8ca051f58ea011a18d2ff31360d365053fe8e744218f06a): Error starting userland proxy: Bind for 0.0.0.0:58087: unexpected error Permission denied

````
(Note the port it tries to bind to)

## Reason (I assume)

Apparently the Windows 10 Hyper-V which is used by Docker Desktop reserves some ports [[1]](https://stackoverflow.com/a/55471778). These can be checked with 

```
$ netsh interface ipv4 show excludedportrange protocol=tcp

Protocol tcp Port Exclusion Ranges

Start Port    End Port
----------    --------
      3939        3939
      5357        5357
     49768       49867
     51901       52000
     56488       56587
     58051       58150
     61271       61370

* - Administered port exclusions.
````
To my understanding the port is randomly assigned by lando. It can be checked with `lando config`, variable `proxyDash`.

## Workaround

[Turning the proxy off](https://docs.devwithlando.io/config/config.html#turning-the-proxy-off):

```
# Edit the config
echo "proxy: 'OFF'" >> ~/.lando/config.yml

# Poweroff lando
lando poweroff

# Reboot an app
lando start SOMEAPP
```` 

Maybe setting a specific value for `proxyDash` in a  `~/.lando/config.yml` other than the reserved ports would help as well.

Lando should avoid choosing these reserved paths.

**Comment by pirog** [comment]
@Clindbergh AFAIK those are not reserved ports by default. if they were i'd suspect we'd have many more comments on this thread. It _looks to me_ like these ports are excluded by some sort of network admin policy.

Not opposed to changing how this works but i think we'd like to see more than one person report similar before we make changes especially given that the port is configurable in the global config
https://docs.devwithlando.io/config/config.html
https://docs.devwithlando.io/config/proxy.html#configuration

**Comment by dashifen-gu** [comment]
This is, actually, happening to me as of this morning.  Showed up, booted my computer, `lando start`ed, and got the same message:

```
ERROR: for proxy  Cannot start service proxy: driver failed programming external connectivity on endpoint landoproxyhyperion5000gandalfedition_proxy_1 (95d0975bcd97ef25164f8ffc92eaa61222dbd9f4b8857c63b9b9f116075941e7): Error starting userland proxy: Bind for 0.0.0.0:58087: unexpected error Permission denied
Encountered errors while bringing up the project.
```

Turning off the proxy was a work around for me, too.

I'm using lando v3.0.0 rc.16 on win10.

**Comment by ow** [comment]
Chiming in to say I have the same problem today too—it looks like a Windows CU did this, but I'm lost as to why. Might be related to the recent Spectre fixes.

**Comment by GMSteuart** [comment]
@ow I ran into this issue today on macOS and from quick google searching I am not entirely sure whether or not Apple has released any fixes recently related to meltdown or spectre.

@pirog I ran into this issue after I needed to do some DNS rebinding on my laptop to get certs working through a company network. I was able to resolve the problem by killing the process that was running on the port, restarting docker, and then continuing as normal. My initial thought is there is that Lando or some other service is caching the port it wants to bind to so any attempts at rebuilding or restarting fail to the fact that Lando kept trying to assign the proxy to port 58087
